docs/src/main/paradox/migrations.md

```diff
@@ -4,6 +4,25 @@
 
 * After being deprecated, the DateTieredCompactionStrategy has been removed.
 
+## Migrations to 0.101 and 0.102
+
+Version 0.101 and 0.102 removes the static column `used`. It is released in two versions to be able to
```

Comment:
maybe just 0.101 and then we add the write in 1.0? Then we can just say go to the latest 0.x before upgrading to 1.9

Reply:
Background is customer request so we can’t wait until 1.0 to be able to completely remove the column

Reply:
:+1: